util/mathutil/math.go

```diff
@@ -54,3 +66,25 @@ func StrLenOfInt64Fast(x int64) int {
 func IsFinite(f float64) bool {
 	return !math.IsNaN(f - f)
 }
+
+// Max choice biggest number from its arguments.
+func Max[v constraints.Ordered](x v, xs ...v) v {
+	max := x
+	for _, n := range xs {
+		if n > max {
+			max = n
+		}
+	}
+	return max
+}
+
+// Min is to return the smallest integer value..
```

Comment:
```suggestion
// Min returns the smallest one from its arguments.
```
Please make sure the comments of `Max` and `Min` follows the similar style.